### PR TITLE
[netatmo] Add translations for config parameters

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Review by clinique — APPROVED** [review]
LGTM

**Comment by jlaur** [comment]
Reason for this PR was found in [this comment](https://github.com/openhab/openhab-addons/pull/12726#discussion_r876023219).

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks. Minor comment added to try to preserve a similar structure to ordinary generated channel type keys.

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/config/config.xml [hunk 1]
```diff
@@ -7,133 +7,133 @@
...
+			<label>@text/config.period.label</label>
+			<description>@text/config.period.measure.description</description>
 			<options>
-				<option value="30min">30 minutes</option>
-				<option value="1hour">1 hour</option>
-				<option value="3hours">3 hours</option>
-				<option value="1day">This day</option>
-				<option value="1week">This week</option>
-				<option value="1month">This month</option>
+				<option value="30min">@text/config.period.30min</option>
```
[the remark this answers is not in the document]
Comment (lolodomo) [reply bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/config/config.xml]: Done

**Review by jlaur — APPROVED** [review]
LGTM, thanks.

**Pull request merged**